lib/Swagger/src/Discussion/Models/PostInput.php

```diff
@@ -63,15 +63,16 @@ class PostInput implements ArrayAccess
       * @var string[]
       */
     protected static $swaggerTypes = array(
-        'body' => 'string',
-        'thread_id' => 'int',
+        'id' => 'int',
         'position' => 'int',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "position" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)